src/app/projects/project-list/project-list.component.ts

```diff
@@ -487,47 +287,55 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     this.getPaginatedProjects(this.tableParams.currentPage);
   }
 
-  getPaginatedProjects(pageNumber) {
-    // Go to top of page after clicking to a different page.
-    window.scrollTo(0, 0);
-    this.loading = true;
+  executeSearch(apiFilters) {
+    this.terms.keywords = apiFilters.keywords;
+    this.tableParams.keywords = apiFilters.keywords;
+    this.filterForAPI = apiFilters.filterForAPI;
+
+    // build filterForUI/URL from the new filterForAPI object
+    this.createFilterForURL();
 
+    this.getPaginatedProjects(this.tableParams.currentPage);
+  }
+
+  createFilterForURL() {
+    this.filterForURL = {};
+    // for each key in filterForAPI
+    Object.keys(this.filterForAPI).forEach(key => {
```

Comment:
Does looping need to occur? I looks like this is just copying the object. Could it be `this.filterForUrl = {...this.filterForApi}`

Reply:
Yep, for sure. Originally I assumed filterForAPI and filterForURL were different (as they exist in two different objects in the original code). Turns out... they're not at all. You can use the objects interchangably. So the right answer here is not so much copying, but just replacing filterForURL to use filterForAPI I think

Reply:
Fixed